nodepool/openstack/clouds.yaml.j2

```diff
@@ -7,6 +7,7 @@ cache:
     port: 5
 
 clouds:
+  __nodepool_clouds_yaml_aws_access_key_id
```

Comment:
why do we need aws credentials in clouds.yaml?

Reply:
ahah, lets me fix that.